app/src/main/dist/config/crate.yml

```diff
@@ -1,15 +1,51 @@
-######################### Crate Configuration Example #########################
+######################### Crate Configuration File ############################
 
 # The default configuration offers the ability to use Crate right away.
 # The purpose of this file is to give operators an overview of the various
 # different configuration settings. To read the full documentation go to
-# <https://docs.crate.io/>
+# <https://crate.io/docs/>
 
-# As mentioned before, you can use Crate right away. For fine-tuning consider
-# to configure Crate as you need or want it to behave.
 # Use this file to fine-tune your Crate cluster. If you have any questions
-# you are very welcome to conact us on our public Community on Slack
-# (https://crate.io/docs/support/slackin/).
+# you are very welcome to contact us on our Community Channel on Slack
+# <https://crate.io/docs/support/slackin/>.
+
+
+################################ Quick Settings ###############################
+
+# Recommended memory settings:
+# - set the environment variable CRATE_HEAP_SIZE to half of your memory
+#   (e.g. 26G, but not more than 30G).
```

Comment:
i'd prefer to keeps things simple here and therefor rather not link to the full explanation about heap.